src/main/scala/com/datastax/sparkstress/SparkCassandraStress.scala

```diff
@@ -109,6 +114,11 @@ object SparkCassandraStress {
           ("spark.cassandra.input.split.size" -> arg.toString))
       } text {"Read input size"}
 
+      opt[Int]('u',"throughput") optional() action { (arg,config) =>
+        config.copy(sparkOps = config.sparkOps +
+          ("spark.cassandra.output.throughput_mb_per_sec" -> arg.toString))
```

Comment:
We can do this but I think this would require us to decide on a predefined number of non-optional params to spark-stress, in order to differentiate between params passed to the stress-tool and params that will be passed along with --conf to spark-submit.

For example do we want to require users to specify totalOps, numPartitions, replication, etc. every time? Unless there is a better way to differentiate the params passed to the run.sh script.